libs/extended/src/Servant/API/Extended.hs

```diff
@@ -116,3 +120,18 @@ instance
 
 instance (RoutesToPaths rest) => RoutesToPaths (ReqBodyCustomError' mods list tag a :> rest) where
   getRoutes = getRoutes @rest
+
+data YAML
+
+instance Accept YAML where
+  contentTypes _ =
+    "application"
+      M.// "yaml"
+      M./: ("charset", "utf-8")
+      NE.:| ["application" M.// "yaml"]
+
+instance {-# OVERLAPPABLE #-} (ToJSON a) => MimeRender YAML a where
+  mimeRender _ = fromStrict . Y.encode
+
+instance {-# OVERLAPPABLE #-} (FromJSON a) => MimeUnrender YAML a where
+  mimeUnrender _ = first show . Y.decodeEither' . toStrict
```

Comment:
Are you sure you want to make this `OVERLAPPABLE`? I think making every instance that would overlap `OVERLAPPING` is the better UX as you wouldn't want to *accidentally* write a new instance for more specific type.